_posts/2020-04-09-virtual-workshop.md

```diff
@@ -11,7 +11,7 @@ This workshop will be held in three online sessions:
 1. **Technical talks** - Short talks about projects that RSEs are working on (Wednesday 4/22, 3:00-4:30 PM EDT)
 1. **Next steps for the US-RSE organization** - Breakout sessions to discuss ideas. What do *you* want from US-RSE? (Thursday 4/23, 12:00-1:30 PM EDT)
 
-A detailed agenda is in the works and will be posted [here]({{ site.baseurl }}/events/2020/2020-04-virtual-workshop/) when ready.
+A detailed agenda is in the works and will be posted [here]({{ site.baseurl }}/events/2020/2020-04-virtual-workshop) when ready.
```

Comment:
Good call, "subject to change" is definitely good to directly state.

@danielskatz if you are previewing on CircleCI, circleci isn't actually a web server, so a file in `folder/index.html` that normally would render at `/folder/` won't work. For any links that you try, you can generally add index.html after it (folder/index.html) to see the page, and have confidence that on an actual web server the index.html will do it's job!

Reply:
- In my browser the link works without the html, but it's probably more portable to add the suffix.
- Will fix wording.

Reply:
@cferenba you shouldn't need to make this change, this is the convention that all links across the site use (that work). The non-working case is just an artifact on circle. I would keep as is, because it's better to be consistent.

Reply:
ok

Reply:
ok - I hadn't looked closely at the rest of the site